src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -17,7 +17,7 @@
 public class DriverDaoImpl implements DriverDao {
     @Override
     public Driver create(Driver driver) {
-        String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
+        String query = "INSERT INTO drivers (name, licenseNumber) VALUES (?, ?)";
```

Comment:
its better to name your rows using the underscore by convention
as like as u did in other tables